api/models/CustomReferralUrl.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  connection: 'photonDb',
+  migrate: 'safe',
+  tableName: 'custom_referral_url',
+  autoCreatedAt: false,
+  autoUpdatedAt: false,
+
+  attributes: {
+    referralUrl: {
+      columnName: 'referral_url',
+      type: 'string',
+      size: 66
```

Comment:
66 feels pretty arbitrary. Maybe just make it a base2 number like 64?

And along these lines, do we have some handling for if the code we generate ends up being longer than the max size? There's always gonna be that one person that puts in 100 As in a row as their first name.